main.js

```diff
@@ -0,0 +1,18 @@
+const nav = document.querySelector('.menu');
+const openButton = document.querySelector('.hamb-icon');
+const closeButton = document.querySelector('.close');
+const item = document.querySelectorAll('.element');
+
+openButton.addEventListener('click', () => {
+  nav.classList.add('visible');
+});
+
+closeButton.addEventListener('click', () => {
+  nav.classList.remove('visible');
+});
+
```

Comment:
- [ ] Great developer! Kindly centralize the horizontal lines (`hr`) as pointed out by the previous reviewer, you are almost there. 👍 

## Your Design
<img width="281" alt="Untitled" src="https://github.com/ulises2607/Portfolio/assets/103386183/d4e4fc38-a4db-4277-947f-3b32e4106bcc">

Figma Design
<img width="141" alt="Untitled" src="https://github.com/ulises2607/Portfolio/assets/103386183/e8f94eb5-af0b-4401-bc8d-17b502b1a43f">